server/cmwell-it/src/it/scala/cmwell/it/APIValidationTests.scala

```diff
@@ -327,28 +327,7 @@ class APIValidationTests extends AsyncFunSpec with Matchers with Inspectors with
       })
     }
 
-//    TODO: should we re-enable this test?
-//    describe("iterator creation") {  //todo this should pass, but fails on Jenkins
-//      import scala.concurrent.duration._
-//
-//      val defaultMax = wsConfig.getInt("webservice.max.search.contexts")
-//      val ttl = 60
-//
-//      val req = Http.get(cmw, List("op" -> "create-iterator", "session-ttl" -> ttl.toString))
-//      val returnCodes = Future.traverse(1 to defaultMax)(_ => req.map(_.status))
-//
-//      (ttl - 3).seconds.fromNow.block
-//      val returnCode = req.map(_.status)
-//
-//      ignore(s"should allow $defaultMax iterators concurently") {
-//        val fi = returnCodes.map(_.count(_ == 200))
-//        Await.result(fi, requestTimeout) should be(defaultMax)
-//      }
-//
-//      ignore(s"should reject the ${defaultMax+1}th request") {
-//        Await.result(returnCode, requestTimeout) should be(400)
-//      }
-//    }

```

Comment:
We need to throttle requests somehow.
This commented out test should be re-enabled sometime in the future.
I say we un-delete it, and UNCOMMENT it, putting in in `ignore` for the time being.